src/Microsoft.ServiceFabric.Actors/Runtime/ActorManager.cs

```diff
@@ -403,7 +403,7 @@ public Task StartLoadingRemindersAsync(CancellationToken cancellationToken)
             ActorTrace.Source.WriteInfoWithId(TraceType, this.traceId, "DeleteActorAsync: Delete call received for actor {0}", actorId);
 
 
-            // Use ActorConcurrencyLock to synchronize with other actor calls.
+            // Uses ActorConcurrencyLock to synchronize with other actor calls.
```

Comment:
This is a code comment telling the follow of the code, this change is not needed.